src/accordion/accordion.spec.ts~RFff54cea.TMP

```diff
@@ -0,0 +1,538 @@
+import {TestBed, ComponentFixture, inject} from '@angular/core/testing';
```

Comment:
Looks like this file made it to this PR by accident, could you please remove it?